test/e2e/network/dns_common.go

```diff
@@ -373,17 +372,9 @@ func createDNSPod(namespace, wheezyProbeCmd, jessieProbeCmd, podHostName, servic
 	dnsPod := e2epod.NewAgnhostPod(namespace, podName, volumes, mounts, nil, "test-webserver")
 	dnsPod.Spec.Containers[0].Name = "webserver"
 
-	querier := e2epod.NewAgnhostContainer("querier", mounts, nil, wheezyProbeCmd)
+	querier := e2epod.NewAgnhostContainer("querier", mounts, nil, probeCmd)
 	querier.Command = []string{"sh", "-c"}
 
-	jessieQuerier := v1.Container{
-		Name:         "jessie-querier",
-		Image:        imageutils.GetE2EImage(imageutils.JessieDnsutils),
-		Command:      []string{"sh", "-c", jessieProbeCmd},
-		VolumeMounts: mounts,
-	}
-
-	dnsPod.Spec.Containers = append(dnsPod.Spec.Containers, querier, jessieQuerier)
```

Comment:
This shouldn’t have been removed entirely, should it?

```
dnsPod.Spec.Containers = append(dnsPod.Spec.Containers, querier)
```

Reply:
yep, found that, too. :)